spec/requests/api/allocations_controller_create_v2_spec.rb

```diff
@@ -0,0 +1,335 @@
+# frozen_string_literal: true
```

Comment:
not sure about this, basically I copied v1 tests and fixed broken ones. I'm wondering what the strategy is regarding v1/v2 tests to avoid duplication cc @willfish ?

Reply:
The strategy is to duplicate and remove everything v1 in 1 or 2 months time.